.gitignore

```diff
@@ -137,3 +137,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# VSCode editor related stuff
+.vscode/
```

Comment:
I have the slightest preference for dev specific things to go in your local global gitignore. Not worth undoing, but think we should avoid intentionally adding to the gitignore of all repos.

Reply:
Learned something new and just in time @marvinmarnold :-) Was thinking whether to add my local docker-compose files (leaner versions with fewer services) for hm-dashboad to hm-dashboard's .gitignore and not and found your comment just in time.

FYI for anyone interested on *nix and mac you can add such patterns to `~/.config/git/ignore`

Reply:
Learned another thing today. That's a `+1` for me :smiley: